.flake8

```diff
@@ -1,3 +1,5 @@
 [flake8]
 exclude = venv
-per-file-ignores = main.py: F401, E402
\ No newline at end of file
+per-file-ignores =
+    main.py: F401, E402
+    .\Google_CLoud_Functions\main.py: E501, F841
```

Comment:
why did you silent those rules?

Reply:
- F401 means 'module imported but unused, although if I delete the import, it doesn't work anymore.
- E402 means 'Module level import not at top of file'  because i first had to define the app variable that is used in the imported modules (endpoints) 
- E501 means  'max-line-length = 120 is exceeded' but this will be deleted since the url's are shorter now.
- F841 means 'Local variable name is assigned to but never used' but is also deleted now